### DISCUSSION TITLE
Kids Channel

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#10] [Show and tell]
here a load o f kids stuff  but since  the streamlinker(fullspeed.tv) dont work for me  for unknow reason  il let you test it and add them 

https://www.youtube.com/c/cartoonnetwork/live
https://www.youtube.com/c/cartoonnetworkasia/live
https://www.youtube.com/c/cartoonnetworkturkiye/live
https://www.youtube.com/c/BRCartoonNetworkvideos/live
https://www.youtube.com/c/CartoonLA/live

https://www.youtube.com/c/wbkids/live
https://www.youtube.com/c/WBKidsInternational/live
https://www.youtube.com/c/WBKidsItaliano/live
https://www.youtube.com/c/WBKidsFran%C3%A7ais/live
https://www.youtube.com/c/WBKidsDeutschland/live
https://www.youtube.com/c/WBKidsEspa%C3%B1a/live
https://www.youtube.com/c/WBKidsBrasil/live

https://www.youtube.com/c/dckids/live
https://www.youtube.com/c/DCKidsFran%C3%A7ais/live
https://www.youtube.com/c/DCKidsBrasil/live
https://www.youtube.com/c/DCKidsLatino/live
https://www.youtube.com/c/DCKidsEspa%C3%B1a/live
https://www.youtube.com/c/DCKidsDeutschland/live

Super Toons TV US
https://www.youtube.com/channel/UCL4xLe_SjcDccfpVeSRFGYQ/live
Super Toons TV FR
https://www.youtube.com/channel/UCE6LObtIIyUymAejzHJ73GA/live
Super Toons TV ES
https://www.youtube.com/channel/UCBO856ZVw88_wjNY2izgu9w/live
Super Toons TV Bahasa
https://www.youtube.com/channel/UClJ1NXIsdoVyfxWg3aLYrsA/live
Super Toons TV Animasyon
https://www.youtube.com/channel/UCm--Y_S6mPZZ5wLbUQ8eATA/live
Super Toons TV アニメ
https://www.youtube.com/channel/UCLOpR7bz6O2BNGi4nF3xfnw/live
Super Toons TV PT/BR
https://www.youtube.com/channel/UCw8lDb58abNexUg4jLGec_A/live
Super Toons TV Tiếng Việt
https://www.youtube.com/channel/UCx5EWcp8bcKx0f1fCgAeyAA/live
Super Toons TV DE
https://www.youtube.com/channel/UCa_0tnyEpU7bny2uSs8qkgQ/live
Super Toons TV PL
https://www.youtube.com/channel/UCuZ4N0diHr2tKmVzRYHdVqw/live

https://www.youtube.com/c/peppapig/live
https://www.youtube.com/c/PeppaPigFR/live
Peppa Pig - Nursery Rhymes and Kids Songs
https://www.youtube.com/channel/UCOYd4Al0Ty89xigE-l9AIog/live
ペッパピッグ ー Peppa Pig
https://www.youtube.com/c/%E3%83%9A%E3%83%83%E3%83%91%E3%83%94%E3%83%83%E3%82%B0/live
https://www.youtube.com/c/TheHomeofPeppaPig/live
https://www.youtube.com/c/PeppaPigOficial/live
https://www.youtube.com/c/OficialPeppaBR/live
https://www.youtube.com/c/PeppaPigITA/live
https://www.youtube.com/c/PeppaPigDE/live
小猪佩奇 中文官方 - Peppa Pig
https://www.youtube.com/channel/UCRi0sw4PDw-C2C7fS9hrx5Q/live

Transformers Kids
https://www.youtube.com/channel/UCPoCHHnaswi9xdXl-JQuD9Q/live
Трансформеры, вперед!
https://www.youtube.com/c/GoTransformers/live
https://www.youtube.com/c/transformersofficial/live
Transformers TV
https://www.youtube.com/channel/UC3lxPjuIiHgRubQsJqhGeEA/live

https://www.youtube.com/c/mylittlepony/live
https://www.youtube.com/c/MyLittlePonyemPortugu%C3%AAsBrasilCanalOficial/live
https://www.youtube.com/c/MyLittlePonyenfran%C3%A7aischa%C3%AEneofficielle/live
My Little Pony ภาษาไทย
https://www.youtube.com/channel/UCOT9qiUmk4mk-B_hN__toTw/live
My Little Pony bahasa Indonesia
https://www.youtube.com/channel/UCDRCl6JTjDKjyquagRWrkgA/live
https://www.youtube.com/c/MyLittlePonyenEspa%C3%B1olLatinoCanalOficial/live
My Little Pony bahasa Melayu
https://www.youtube.com/channel/UC3wU_P1Q386FSZoYTviOKfg/live
My Little Pony Polski
https://www.youtube.com/channel/UC5MtckkZTJRHZDbpPo3-vjw/live
https://www.youtube.com/c/MyLittlePonyDeutschOffiziellerKanal/live
My Little Pony 小馬寶莉 官方頻道
https://www.youtube.com/channel/UCJVDGk1GqIq98bobbaszhiA/live
My Little Pony 마이 리틀 포니 - 한국어
https://www.youtube.com/channel/UC2RETnhkVhRbnUjjZX4gH_g/live

https://www.youtube.com/c/BeanoOfficial/live

https://www.youtube.com/barneyandfriends/live

https://www.youtube.com/c/AntiksOfficial/live

https://www.youtube.com/c/Hasbro/live

https://www.youtube.com/c/GIJOEOfficialChannel/live

https://www.youtube.com/c/Peacockjr/live

https://www.youtube.com/c/strawberryshortcakeofficial/live

https://www.youtube.com/user/disneyjunior/live

https://www.youtube.com/c/DisneyPrincess/live

https://www.youtube.com/c/cartoonito/live

https://www.youtube.com/c/blazeandthemonstermachines/live

https://www.youtube.com/c/hotwheels/live

https://www.youtube.com/c/LocoNutsCartoonVideos/live

https://www.youtube.com/c/NetflixJr/live

https://www.youtube.com/c/netflixafterschool/live

https://www.youtube.com/c/NickelodeonCartoonUniverse/live

https://www.youtube.com/user/OfficialPawPatrol/live

https://www.youtube.com/c/officialPeterRabbit/live

https://www.youtube.com/c/robocarpoli/live

https://www.youtube.com/sesame/live

https://www.youtube.com/c/TattyandMisifu/live

https://www.youtube.com/c/tayo/live

https://www.youtube.com/c/LoudCasa/live

https://www.youtube.com/c/thomasandfriends/live

https://www.youtube.com/c/ZIPZIPEnglishOfficial/live

https://www.youtube.com/c/SimonSuperRabbitEnglish/live

Magic Cartoon Animals! - Morphle TV
https://www.youtube.com/channel/UClMk-uURdArGhz1lCfSRn6Q/live

Tiddlytubbies
https://www.youtube.com/channel/UCvCxn5cIlSSnQy26DGxp1Sg/live
https://www.youtube.com/c/teletubbies/live

https://www.youtube.com/c/PowerRangersOfficial/live
https://www.youtube.com/c/PowerRangerspourlesEnfantsCha%C3%AEneOfficielle/live
https://www.youtube.com/c/PowerRangersparaNi%C3%B1osCanalOficial/live

**Comment by princo17** [comment]
merci

**Comment by BellezaEmporium** [comment]
Most of them are simply geo-locked, that's why it doesn't work.

**Comment by princo17** [comment]
ok

**Comment by fotvnetwork** [comment]
Thank you

**Comment by SSyar** [comment]
Fantastic.,

**Comment by ThetaX55** [comment]
Thanks. 
Any other channels you have that are on youtube? 
Movie's, super hero shows

**Comment by Sphinxroot** [comment]
as exemple  some might not be 24/7

TWiT
https://www.youtube.com/channel/UCwY9B5_8QDGP8niZhBtTh8w/live

US Local

all us very local
https://www.youtube.com/results?search_query=very+local&sp=EgJAAQ%253D%253D

News 12
https://www.youtube.com/channel/UCFhbonoFqxg2b9FMRzYAwog/live
ABC7 SWFL
https://www.youtube.com/channel/UCq9e_hCv2jvgck8WowW1NXg/live
WBRA-TV PBS Blue Ridge
https://www.youtube.com/channel/UClfc1_I5AVnnRAvIRpwLXBQ/live
Blue Ridge PBS Echo TV
https://www.youtube.com/channel/UCPq-p0HbZfG4ADIlvqGJIZA/live
Iowa PBS
https://www.youtube.com/channel/UCHHphhxRHZpMS96kUr4DuwA/live
NBC2 News
https://www.youtube.com/channel/UCIYVU7fPs_n6gJIBso3ryeg/live

Doc And TV Show:

CNBC
https://www.youtube.com/channel/UCvJJ_dzjViJCoLf5uKUTwoA/live
Travel Channel
https://www.youtube.com/channel/UCJUAqHnkDX15IvFoXIGTm2A/live
Quest TV
https://www.youtube.com/channel/UC2hMKdWmTBVJ_PnD4Utubbw/live
Discovery UK
https://www.youtube.com/channel/UC9cGX5x0_EXkdfmWlvPbu3g/live
TLC UK
https://www.youtube.com/channel/UC2vlpX8sNDBmPcY_1_QSGJg/live
TV Rerun Club by MGM  (1 of 4 Stream)
https://www.youtube.com/channel/UC3ndD2cB161KQjeMbUXNbQg/live
Chateau De Bailey
https://www.youtube.com/channel/UCWfzNUssnBhBqxc_DFPsV6Q/live
COPSTV
https://www.youtube.com/channel/UCfJsOa7m1zKMS9T2dvIljfQ/live
The Dick Cavett Show
https://www.youtube.com/channel/UCFC8Vt3FY_7svm_SOBK5aIQ/live
Nat Geo WILD
https://www.youtube.com/channel/UCDPk9MG2RexnOMGTD-YnSnA/live
National Geographic
https://www.youtube.com/channel/UCpVm7bg6pXKo1Pr6k5kxG9A/live
Cold Case Detective
https://www.youtube.com/channel/UCdjslyNQupPSFxK_mSPcG-g/live
Spark
https://www.youtube.com/channel/UCMV3aTOwUtG5vwfH9_rzb2w/live
Real Stories
https://www.youtube.com/channel/UCu4XcDBdnZkV6-5z2f16M0g/live
Expoza Travel
https://www.youtube.com/channel/UC3o_gaqvLoPSRVMc2GmkDrg/live
Daily Dose Documentary
https://www.youtube.com/channel/UCuYZAnKoZYXAKKpsi_-90Tg/live


News :

FOX Weather
https://www.youtube.com/channel/UC1FbPiXx59_ltnFVx7IxWow/live
USA TODAY
https://www.youtube.com/channel/UCP6HGa63sBC7-KHtkme-p-g/live
Newsmax
https://www.youtube.com/channel/UCx6h-dWzJ5NpAlja1YsApdg/live
ABC News
https://www.youtube.com/channel/UCBi2mrWuNuyYy4gbM6fU18Q/live
NBC News
https://www.youtube.com/channel/UCeY0bbntWzzVIaj2z3QigXg/live
Bloomberg Originals
https://www.youtube.com/channel/UCUMZ7gohGI9HcU9VNsr2FJQ/live
Al Jazeera English
https://www.youtube.com/channel/UCNye-wNBqNL5ZzHSJj3l8Bg/live
CNA
https://www.youtube.com/channel/UC83jt4dlz1Gjl58fzQrrKZg/live
Channels Television (1 of 2 Stream)
https://www.youtube.com/channel/UCEXGDNclvmg6RW0vipJYsTQ/live
DW News English
https://www.youtube.com/channel/UCknLrEdhRCp1aegoMqRaCZg/live
FRANCE 24 English
https://www.youtube.com/channel/UCQfwfsi5VrQ8yKZ-UWmAEFg/live
Euronews English
https://www.youtube.com/channel/UCSrZ3UV4jOidv8ppoVuvW9Q/live
Sky News UK
https://www.youtube.com/channel/UCoMdktPbSTixAyNGwb-UYkQ/live
Africanews English
https://www.youtube.com/channel/UC1_E8NeF5QHY2dtdLRBCCLA/live
ABC News (Australia)
https://www.youtube.com/channel/UCVgO39Bk5sMo66-6o6Spn6Q/live
TRT World
https://www.youtube.com/channel/UC7fWeaHhqgM4Ry-RMpM2YYw/live
NHK WORLD-JAPAN
https://www.youtube.com/channel/UCSPEjw8F2nQDtmUKPFNF7_A/live
GBNews
https://www.youtube.com/channel/UC0vn8ISa4LKMunLbzaXLnOQ/live
Yahoo Finance
https://www.youtube.com/channel/UCEAZeUIeJs0IjQiqTCdVSIg/live
KTN News
https://www.youtube.com/channel/UCKVsdeoHExltrWMuK0hOWmg/live
India Today (1 of 12 streams)
https://www.youtube.com/channel/UCYPvAwZP8pZhSMW8qs7cVCw/live
LiveNOW from FOX
https://www.youtube.com/channel/UCJg9wBPyKMNA5sRDnvzmkdg/live
TVC News Nigeria (1 of 2 Stream)
https://www.youtube.com/channel/UCgp4A6I8LCWrhUzn-5SbKvA/live
WION English
https://www.youtube.com/channel/UC_gUM8rL-Lrg6O3adPW9K1g/live
NDTV
https://www.youtube.com/channel/UCZFMm1mMw0F81Z37aaEzTUA/live


Sports:
Monster Jam
https://www.youtube.com/channel/UCngNXEVdMZge3OpnD8Ca0Lw/live
Monster Energy Supercross
https://www.youtube.com/channel/UCkaNo2FUEWips2z4BkOHl6Q/live

################

hint:
https://www.youtube.com/results?search_query=STRINGHERE&sp=EgJAAQ%253D%253D

STRINGHERE To anything you want
country by:  name  or tag .ca .us ....
subject : Sports doc movies space kids News radio music television channel ...
use any language to search
some are geoblocked and can be found via vpn
